compass/landice/tests/ensemble_generator/ensemble_manager.py

```diff
@@ -63,8 +65,24 @@ def run(self):
             # Get step object from 'steps' dictionary
             runStep = self.test_case.steps[run]
             os.chdir(runStep.work_dir)
-            # TODO: assess if this run is unrun, partially run, or complete,
-            # and adjust accordingly
+
+            # Skip run if an error is detected
+            err_files = glob.glob('log.landice.*.err')
```

Comment:
I agree their is some fragility here but I wanted to be conservative so that if there are any issues with a run it doesn't get automatically relaunched.  A user could inspect that run and deal with it.

In general one shouldn't change code versions in an ensemble.  I've added a note to the users guide pointing that out.